applications/minotari_node/src/http/server.rs

```diff
@@ -74,7 +75,10 @@ impl<S: BaseNodeWalletQueryService> Server<S> {
             .route("/transactions", get(handler::transaction_query::handle::<B>))
             .route("/sync_utxos_by_block", get(handler::sync_utxos_by_block::handle::<B>))
             .route("/get_utxos_by_block", get(handler::get_utxos_by_block::handle::<B>))
-            .route("/json_rpc", post(handler::json_rpc::handle::<B>))
+            .route(
+                "/json_rpc",
+                post(handler::json_rpc::handle::<B>).layer(DefaultBodyLimit::max(4096)),
```

Comment:
Let's up it to what grpc uses
```suggestion
                post(handler::json_rpc::handle::<B>).layer(DefaultBodyLimit::max(8192)),
```